lib/events/index.js

```diff
@@ -254,7 +254,11 @@ export default function (self) {
     setScrollBoxSize();
     self.scrollBox.scrollWidth = dataWidth - self.scrollBox.width;
     self.scrollBox.scrollHeight = dataHeight - self.scrollBox.height;
-    self.scrollBox.widthBoxRatio = self.scrollBox.width / dataWidth;
+    if (self.frozenColumn > 0)
```

Comment:
Same comment as above; I'd prefer braces around anything but the simplest of `if` statements. For instance:

```js
if (foo === false) return;
```

is fine. But

```js
if (foo === false) bar = quux + bar === 123 ? getSomething() : getSomethingElse();
else return doSomethingAndReturnIt();
```
is (IMHO) already a bit risky. What do you think?

Reply:
ok, that's great